rom/dev/src/main.rs

```diff
@@ -70,10 +70,26 @@ pub extern "C" fn rom_entry() -> ! {
     if !cfg!(feature = "no-cfi") {
         cprintln!("[state] CFI Enabled");
         CfiCounter::reset(&mut env.trng);
+        CfiCounter::reset(&mut env.trng);
+        CfiCounter::reset(&mut env.trng);
     } else {
         cprintln!("[state] CFI Disabled");
     }
 
+    // Check if TRNG is correctly sourced as per hw config.
+    match env.trng {
```

Comment:
Does this compile to a single branch to then assert three times that the configuration is correct? If so I think it should be split out into three match statements so it's less easy to glitch past the single branch to the incorrect match. (Or possibly a function that does the verification, where that function is called several times)

Reply:
What do you think of https://github.com/chipsalliance/caliptra-sw/pull/979/commits/414433043c1f4daaace90e64ba19a4daa01f882e ?

Reply:
Merged that change into this PR. Thoughts?

Reply:
Yeah, I think that's likely to look good when compiled. I'll need to have a look at the compiled version of this when I get a chance.